### ISSUE TITLE
Dash installation instructions needlessly require dash-renderer and plotly install

### ISSUE THREAD

**Issue by felixvelariusbos** [plotly/dash-docs#161]
I recently re-installed Dash on a new computer, and noticed that when I `pip install dash`, it automatically installs dash-renderer and plotly; I don't need to do that.

It's not the worst thing in the world, but the installation page doesn't need to instruct me to install dash-renderer or plotly. Unless there's some edge case I can't think of where it's needed to know all the relevant Dash libraries, could the installation instructions be changed to reflect this? 3 packages is much less intimidating than 5. :)

**Comment by rmarren1** [comment]
Are you talking aboue [here](https://dash.plot.ly/installation)? This will give you the most up to date packages. Sometimes installing `dash` will give you the version of `dash-renderer` required for that `dash` version, but we could have updated `dash-renderer` with more features in the meanwhile.

For example, installing `dash` now gives you 
https://github.com/plotly/dash/blob/7219b3d9f827e95bbbace801aa955b8c21ff8ffe/requirements.txt#L6
when you probably want all the goodies in `0.13.1`

**Comment by T4rk1n** [comment]
Look like `plotly` and `dash_renderer` were added in plotly/dash#252 so now they install the latest version when installing dash.

**Comment by felixvelariusbos** [comment]
@rmarren1 Yes sorry for being unclear; that's the page I was talking about.

**Comment by rmarren1** [comment]
Oh I thought we pulled dependencies from `requirements.txt`.

It makes sense for `dash-renderer` to be removed, since users will never directly interact with it when making a Dash app.

**Comment by chriddyp** [comment]
There is still some formatting to do, but this is how it could look:
![image](https://user-images.githubusercontent.com/1280389/44819267-3cf8b880-abba-11e8-9db0-5a02f9a87c33.png)

One downside of this is that the installation page is now a lot longer. If the text is appropriately de-emphasized, perhaps it's OK. 

Thoughts?

**Comment by felixvelariusbos** [comment]
I hadn't thought of it being a handy way to see the current versions, so I see you're point there.

Since the instructions are telling users to use pip anyways, they can also just use `pip list` or `pip freeze` to check versions. However showing the __version__ variable is also doubling to show some functionality of the library (I can think of some use cases where I would check dash versions before doing some logic). So either/or there. :)

I like links to the changelogs. I often visit those pages so having an easy link is helpful. My only request for change is to, instead of using the raw URL, use a label like [dash-renderer version 0.12.0](https://github.com/plotly/dash-renderer/blob/master/CHANGELOG.md).

**Comment by chriddyp** [comment]
> Since the instructions are telling users to use pip anyways, they can also just use pip list or pip freeze to check versions

Yeah, I've found that in helping users, sometimes the version that they _think_ that they are installing with `pip` isn't the one that's actually being used when they run the app with `python` - either because they haven't activated their virtualenv or they're actually doing something like `pip`/`python3`. So, I like having a way for a user to see the versions that are getting run _during_ the runtime

**Comment by chriddyp** [comment]
Alright, here's the latest version: 
![image](https://user-images.githubusercontent.com/1280389/45003803-21275500-afb4-11e8-9458-b0bdf1cdfd9a.png)
